sympy/core/add.py

```diff
@@ -437,12 +436,6 @@ def as_numer_denom(self):
         for f in expr.args:
             ni, di = f.as_numer_denom()
             nd[di].append(ni)
-        # put infinity in the numerator
```

Comment:
what does `(1 + Pow(0,-1,evaluate=False)).as_numer_denom()` with this removal?

Reply:
`(zoo, 1)`, same as in the latest release.